### PR TITLE
hwdef: add support for Sierra-F1 AP

### PR DESCRIPTION
**Pull request by MallikarjunSE**
Hello @tridge 
Can you check this? This adds support for Sierra-F1 AP with Ethernet. 
Thank you!

### CODE REVIEW

**Review by Hwurzburg — CHANGES_REQUESTED** [review]
Needs README.md

**Comment by MallikarjunSE** [comment]
> Needs README.md

Hey @Hwurzburg 
Updated.

File: libraries/AP_HAL_ChibiOS/hwdef/Sierra-F1/README.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+# Daughter board:
+ -Ethernet PHY
+ -ICM42688/ICM45686/ADIS16507
+ -DPS310
+ -More PWMs with ALT function
+
+## Physical spec:
+ -Weight: 15g, Dimension: 35x45mm
+
+## UART Mapping
```
Comment (Hwurzburg) [inline libraries/AP_HAL_ChibiOS/hwdef/Sierra-F1/README.md:25]: we like to specify which have DMA..its important when using a port for RC

File: libraries/AP_HAL_ChibiOS/hwdef/Sierra-F1/README.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+ - 6 PWM outputs with DSHOT capability
+ - Dual FDCAN
+ - RCIN port
+ - Dual power input with 2xI2C
+
+# Daughter board:
+ -Ethernet PHY
+ -ICM42688/ICM45686/ADIS16507
+ -DPS310
+ -More PWMs with ALT function
```
Comment (Hwurzburg) [inline libraries/AP_HAL_ChibiOS/hwdef/Sierra-F1/README.md:20]: what ALT function? need more info for setup and capabilities...how many PWMs,groups,etc.? if this has a co-processor doesn't it need to be specified somehow in hwdef?

File: libraries/AP_HAL_ChibiOS/hwdef/Sierra-F1/README.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+## PWM Output
+
+The Sierra-F1 supports up to 6 PWM outputs and DSHOT. Additional PWMs can be accessed via daugter board with ALT function.
+The 6 PWM outputs are in 2 groups:
+
+ - PWM 1, 2, 3 and 4 in group1
+ - PWM 5, 6 in group2
+
+# PWM channels
```
Comment (Hwurzburg) [inline libraries/AP_HAL_ChibiOS/hwdef/Sierra-F1/README.md:52]: did you consider enabling BDShot on some or all of the outputs?

File: libraries/AP_HAL_ChibiOS/hwdef/Sierra-F1/README.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
+# Sierra-F1 Autopilot by Sierra Aerospace
```
Comment (Hwurzburg) [inline libraries/AP_HAL_ChibiOS/hwdef/Sierra-F1/README.md:1]: Need a link to where this can be bought

**Review by Hwurzburg — CHANGES_REQUESTED** [review]

File: libraries/AP_HAL_ChibiOS/hwdef/Sierra-F1/README.md (hunk 4 above)
Comment (MallikarjunSE) [reply libraries/AP_HAL_ChibiOS/hwdef/Sierra-F1/README.md:1]: Website is in making. Rest details will be updated as soon as we finish testing. Still the testing is going on.
Thank you!

**Comment by Hwurzburg** [comment]
Please do not mark review comments as resolved when you have neither responded nor made any changes

File: libraries/AP_HAL_ChibiOS/hwdef/Sierra-F1/README.md (hunk 3 above)
Comment (Hwurzburg) [reply libraries/AP_HAL_ChibiOS/hwdef/Sierra-F1/README.md:54]: author states he does not care about supporting BDShot at this time

File: libraries/AP_HAL_ChibiOS/hwdef/Sierra-F1/README.md (hunk 2 above)
Comment (Hwurzburg) [reply libraries/AP_HAL_ChibiOS/hwdef/Sierra-F1/README.md:22]:
```suggestion
 -More PWMs with ALT function
```
```suggestion
 -More PWMs with ALT function (this is a future planned addition, not present now)
```

**Pull request closed without merging**